acme/acme/client.py

```diff
@@ -755,6 +799,19 @@ def _assess_tos(tos):
                 regr = regr.update(terms_of_service_agreed=True)
             return self.client.new_account(regr)
 
+    def update_registration(self, regr, update=None):
```

Comment:
Yes, because `__getattr__` only gets `vars` from `self.client` and `dir` from `ClientBase`. This is an intentional check on us to make sure we're properly handling any differences. Even with putting the meat into `ClientBase`, we'll still want this method so that when it's a clientv2 we pass it through.

Reply:
although I guess now that it is in fact in `ClientBase` we're cool to call it (it wasn't on an earlier pass)